dev/set_matrix.py

```diff
@@ -449,7 +449,7 @@ def main(args):
     print(json.dumps(args, indent=2))
     matrix = generate_matrix(args)
     is_matrix_empty = len(matrix) == 0
-    matrix = sorted(matrix, key=lambda x: x.job_name)
+    matrix = sorted(matrix, key=lambda x: (x.name, x.category, x.version))
```

Comment:
The current order looks like this:

<img width="332" alt="image" src="https://user-images.githubusercontent.com/17039389/206113748-cd64f6e2-ec6a-43bb-8ec7-25408001c188.png">